WordPress/Sniffs/PHP/YodaConditionsSniff.php

```diff
@@ -46,6 +46,8 @@ public function process_token( $stackPtr ) {
 		$beginners   = PHP_CodeSniffer_Tokens::$booleanOperators;
 		$beginners[] = T_IF;
 		$beginners[] = T_ELSEIF;
+		$beginners[] = T_RETURN;
+		$beginners = array_merge( $beginners, PHP_CodeSniffer_Tokens::$assignmentTokens );
```

Comment:
Oh.. also, all the `PHP_CodeSniffer_Tokens` arrays have the same constant as both key as value. If that pattern was applied here for the three additional tokens, the `array_merge()` can be replaced with the faster `+` union operator.